ui/src/components/common/utils.ts

```diff
@@ -144,6 +144,14 @@ export function getPartyLabel(partyId: string, parties: IPartyInfo[]) {
     return { party: partyInfo?.party || partyId , label: partyInfo?.label.substring(0, partyInfo.label.lastIndexOf('|')) || partyId}
 }
 
+export const formatParty = (party: string): string => {
+  return party.split("::")[0]
+}
+
+export const makePartyLabel = (party: string, name: string | undefined, role: string): string => {
```

Comment:
can also be `name?: string`

Reply:
I had tried that and it gave me "A required parameter cannot follow an optional parameter"...which makes sense, but I liked the order I had so kept it like this